sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2992,6 +2991,16 @@ private void migrateVaccinationInfo() throws SQLException {
 			formatRawResultDate(caseInfo, 9);
 			formatRawResultDate(caseInfo, 10);
 			formatRawResultDate(caseInfo, 11);
+
+			if (DataHelper.isNullOrEmpty((String) caseInfo[12])) {
```

Comment:
Replace with formatRawResultInteger -> I'll create a follow-up refactoring issue for this